ziti/cmd/ziti/util/rest.go

```diff
@@ -377,13 +377,13 @@ func GetArtifactoryPath(staging bool, appName string, branch string) string {
 	var path string
 	if staging {
 		path = "ziti-staging/"
-	} else if branch == "master" {
+	} else if branch == "main" {
 		path = "ziti-release/"
 	} else {
 		path = "ziti-snapshot/" + branch + "/"
 	}
 	// Special-case the source-repo when dealing with ziti-prox-c
-	if branch == "master" && appName == c.ZITI_PROX_C {
+	if branch == "main" && appName == c.ZITI_PROX_C {
```

Comment:
this is no longer getting updates.

ziti-prox-c is published to github relelases: https://github.com/openziti/ziti-sdk-c/releases/latest

Reply:
yes, I also updated `ziti use` to pull from GH. Could probably pull this bit out.